stock_picking_transfer_lot_autoassign/wizards/stock_backorder_confirmation.py

```diff
@@ -2,20 +2,18 @@
 # Copyright 2017 Sergio Teruel <sergio.teruel@tecnativa.com>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from openerp import api, models
+from odoo import api, models
 
 
 class StockBackorderConfirmation(models.TransientModel):
     _inherit = 'stock.backorder.confirmation'
 
     @api.multi
     def process_assign_ordered_qty(self):
```

Comment:
@sergio-teruel What is the purpose of this feature? I do not understand it. To me the backorder wizard is not the place to assign automatically all quantities to the existing picking. In any case the new module `stock_pack_operation_auto_fill` does what you seem to require more elegantly. https://github.com/OCA/stock-logistics-workflow/pull/361

I propose to remove this feature from the current module.

Reply:
@sergio-teruel what do you think about this?